### PR TITLE
Update to Lucien

### PR DESCRIPTION
**Pull request by ddbb07**
- Updated clean message
- Removed unecessary active conditions for patch messages
- Added more patch messages
- Removed old patch message

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -5070,19 +5070,60 @@ plugins:
...
+        condition: 'active("ccbgssse006-stendarshammer.esl") and not active("Lucien-StendarrsHammer-Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Creation Club: Tundra Homestead' ]
+        condition: 'active("cceejsse001-hstead.esm") and not active("Lucien-TundraHomestead-Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Creation Club: Umbra' ]
+        condition: 'active("ccbgssse016-umbra.esm") and not active("Lucien-Umbra-Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Insects Begone' ]
+        condition: '(active("Insects Begone - Chaurus.esp") or active("Insects Begone - Spiders.esp") or active("Insects Begone - Spiders and Chaurus.esp")) and not active("Lucien-InsectsBegone-Patch.esp")'
```
Comment (MacSplody) [inline masterlist.yaml]: A regex would be easier in this case.
```suggestion
        condition: 'active("Insects Begone - (Chaurus|Spiders|Spiders and Chaurus)\.esp") and not active("Lucien-InsectsBegone-Patch.esp")'
```

Comment (ddbb07) [reply masterlist.yaml]: I'll change it

**Review by MacSplody — APPROVED** [review]
The rest looks good.

**Comment by ddbb07** [comment]
> The rest looks good.

Are you going to merge it

**Comment by MacSplody** [comment]
> Are you gonna merge it

Yeah, just adding some info to the commit message.

**Comment by ddbb07** [comment]
> > Are you gonna merge it
> 
> Yeah, just adding some info to the commit message.

Ok, sorry for the impatience

**Comment by MacSplody** [comment]
> Ok, sorry for the impatience

No worries, thanks.

**Pull request merged**